src/testdata/omaha-response-example.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<response protocol="3.0" server="nebraska">
+  <daystart elapsed_seconds="0">
+  </daystart>
+  <app appid="{e96281a6-d1af-4bde-9a0a-97b76e56dc57}" status="ok">
+    <updatecheck status="ok">
+      <urls>
+        <url codebase="https://staging.updateservice.flatcar-linux.net/v1/update/" />
+      </urls>
+      <manifest version="3732.0.0">
+        <packages>
+          <package name="flatcar_production_update.gz" hash_sha256="fa7392bef3a4fa90268b345241d6a3052237ce7d2fa41b4ed0f5e770f3f8dab5" size="382137917" required="false"></package>
```

Comment:
Codebase is also wrong, this should be the payload host, not the Omaha endpoint.

How about using this here from the public server for testing:
```
<?xml version="1.0" encoding="UTF-8"?>
<response protocol="3.0" server="nebraska"><daystart elapsed_seconds="0"></daystart><app appid="e96281a6-d1af-4bde-9a0a-97b76e56dc57" status="ok"><ping status="ok"></ping><updatecheck status="ok"><urls><url codebase="https://update.release.flatcar-linux.net/amd64-usr/3732.0.0/"></url></urls><manifest version="3732.0.0"><packages><package name="flatcar_production_update.gz" hash="I6DGNAOrJRUxPbCuLw+AITfiVMo=" size="382137917" required="true"></package><package name="oem-azure.gz" hash="wepxwEV9L2SS1l/ycEZSqWM3dDc=" hash_sha256="3aed3129de50b959a97e4913ba485bd60e72d2bb6aa377d5ed404103f0680043" size="40897503" required="false"></package><package name="oem-qemu.gz" hash="+06iWWI6gaSlcIHV7pjIEJrE9CA=" hash_sha256="8ab630ee4079ecd5f8f512c05b44fec5e4f8db844db916c67c8d54a575cfe506" size="2282" required="false"></package></packages><actions><action event="postinstall" sha256="stLG3U/o4Ar8TMHFwT/RWB0iNkaWOO6QtLrq6+AHBbA=" DisablePayloadBackoff="true"></action></actions></manifest></updatecheck><event status="ok"></event></app></response>
```

Reply:
FYI, this is the command that got the above response: `curl -d @<(echo '<?xml version="1.0" encoding="UTF-8"?> <request protocol="3.0" version="update_engine-0.4.10" updaterversion="update_engine-0.4.10" installsource="scheduler" ismachine="1">     <os version="Chateau" platform="CoreOS" sp="2512.2.0_x86_64"></os>     <app appid="e96281a6-d1af-4bde-9a0a-97b76e56dc57" version="1.2.3" track="alpha" bootid="{965fb4c5-ad3e-4eb7-a4c2-ca0c0e31ec84}" oem="ami" oemversion="0.1.1-r1" alephversion="1688.5.3" machineid="abce671d61774703ac7be60715220bfe" lang="en-US" board="amd64-usr" hardware_class="" delta_okay="false" >         <ping active="1"></ping>         <updatecheck></updatecheck>         <event eventtype="3" eventresult="1"></event>     </app> </request>')  -H 'Content-Type: text/xml' https://public.update.flatcar-linux.net/v1/update/`

Reply:
Done, thanks